.gitignore

```diff
@@ -1,5 +1,2 @@
-/.idea
-/composer.lock
-/nbproject
```

Comment:
IDE files should be ignored globally once on one's pc, not in every project for every IDE.

https://stackoverflow.com/a/7335487/1348344

Reply:
I see